frontend_server/lib/server.dart

```diff
@@ -148,29 +150,17 @@ class _FrontendCompiler implements CompilerInterface {
       ..sdkRoot = sdkRoot
       ..strongMode = false
       ..target = new FlutterTarget(new TargetFlags())
-      ..onError = (CompilationMessage message) {
-        final StringBuffer outputMessage = new StringBuffer()
-          ..write(_severityName(message.severity))
-          ..write(': ');
-        if (message.span != null) {
-          outputMessage.writeln(message.span.message(message.message));
-        } else {
-          outputMessage.writeln(message.message);
-        }
-        if (message.tip != null) {
-          outputMessage.writeln(message.tip);
-        }
-        _outputStream.write(outputMessage);
-      };
+      ..reportMessages = true;
+
     Program program;
     if (options['incremental']) {
       _generator = generator != null
-        ? generator
-        : await IncrementalKernelGenerator.newInstance(
-          compilerOptions, Uri.base.resolve(_filename),
-          useMinimalGenerator: true
-        );
-      final DeltaProgram deltaProgram = await _generator.computeDelta();
+          ? generator
+          : await IncrementalKernelGenerator.newInstance(
+              compilerOptions, Uri.base.resolve(_filename),
+              useMinimalGenerator: true);
```

Comment:
this would be a good place to leave a trailing comma to make dartfmt use its more readable style